roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -38,7 +38,7 @@
     - item.value.partitions|count == 0
     - item.value.holders|count == 0
     - osd_auto_discovery
-    - dmcrypt_journal_collocation
+    - osd_scenario == 'dmcrypt_journal_collocation'
```

Comment:
Shouldn't this be ``osd_scenario == 'journal_collocation'``? You'd need to check the ``dmcrypt`` variable as well.